src/data/faq.json

```diff
@@ -54,6 +54,14 @@
                 "title": "Common Issues",
                 "id": "most_asked",
                 "accordion": [
+                    {
+                        "title": "[Apple/iOS]: Why do I see a screen which tells me to update my operating system?",
+                        "anchor": "update_iOS",
+                        "active": true,
+                        "textblock": [
+                            "This screen is shown on iPhones running an iOS version between iOS 13.5 and (including) iOS 13.6.1. On these iOS Versions, the used version 2 of the Exposure Notification Framework is not availibale, therefore it's necessary to update to iOS 13.7 or newer, to continue on using the Corona-Warn-App. More information can be found in <a href='../faq/#ios135' target='_blank' rel='noopener noreferrer'>this</a> FAQ entry."

```

Comment:
the used version 2 => the necessary version 2

availibale => available

continue on using => continue to use